tests/perf-test/ReportActionsList.perf-test.js

```diff
@@ -0,0 +1,191 @@
+import {measurePerformance} from 'reassure';
+import Onyx from 'react-native-onyx';
+import {screen, fireEvent} from '@testing-library/react-native';
+import ReportActionsList from '../../src/pages/home/report/ReportActionsList';
+import ComposeProviders from '../../src/components/ComposeProviders';
+import OnyxProvider from '../../src/components/OnyxProvider';
+import {ReportAttachmentsProvider} from '../../src/pages/home/report/ReportAttachmentsContext';
+import {WindowDimensionsProvider} from '../../src/components/withWindowDimensions';
+import {LocaleContextProvider} from '../../src/components/LocaleContextProvider';
+import * as LHNTestUtils from '../utils/LHNTestUtils';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+import wrapOnyxWithWaitForBatchedUpdates from '../utils/wrapOnyxWithWaitForBatchedUpdates';
+import PusherHelper from '../utils/PusherHelper';
+import variables from '../../src/styles/variables';
+import {ActionListContext, ReactionListContext} from '../../src/pages/home/ReportScreenContext';
+
+const ONYXKEYS = {
+    PERSONAL_DETAILS_LIST: 'personalDetailsList',
+    SESSION: 'session',
+    COLLECTION: {
+        REPORT: 'report_',
+        REPORT_ACTIONS: 'reportActions_',
+    },
+    NETWORK: 'network',
+};
+
+jest.mock('@react-navigation/native', () => {
+    const actualNav = jest.requireActual('@react-navigation/native');
+    return {
+        ...actualNav,
+        useRoute: () => jest.fn(),
+    };
+});
+
+jest.mock('../../src/hooks/useLocalize', () =>
```

Comment:
Just wonder if we should mock this in jest/setupAfterEnv.js.
Looks like we can mock localize kinda globally to have same behavior for each test

Reply:
Turned out that this mock was unnecessary here, so I removed.